Code/GraphMol/Fingerprints/Fingerprints.cpp

```diff
@@ -392,20 +396,16 @@ ExplicitBitVect *RDKFingerprintMol(
               aIdx = atomsInPath.find_next(aIdx);
             }
           }
+          if (bitInfo) {
```

Comment:
This took me a bit to grok, but this ensures you are adding the current path to ALL bits that are set, not just the first, correct?

Reply:
I'm happy to commit, btw.

Reply:
Yeah, the change is to make sure that the current path is added to the bitInfo structure for all bits.